src/overlay/SurveyMessageLimiter.cpp

```diff
@@ -24,6 +24,12 @@ SurveyMessageLimiter::addAndValidateRequest(
     SurveyRequestMessage const& request,
     std::function<bool()> onSuccessValidation)
 {
+    if (request.commandType != TIME_SLICED_SURVEY_TOPOLOGY)
```

Comment:
Will this if ever be true? Given that we're dropping support at the XDR level, the decoder should always reject any unsupported types. Perhaps this should be a sanity check assert instead?

Reply:
Good point. I changed this to an assert. I was thinking someone could craft a message with `commandType` set to something other than `TIME_SLICED_SURVEY_TOPOLOGY`, but you're right that the XDR decoder should reject that as an invalid enum.